### PR TITLE
Remove deprecated API from fastboot service

### PR DESCRIPTION
**Pull request by kratiahuja**
Removes some deprecations that were targeted for 0.9.9.

cc: @rwjblue @stefanpenner

### CODE REVIEW

File: app/services/fastboot.js [hunk 1]
```diff
@@ -75,12 +73,6 @@ const FastBootService = Ember.Service.extend({
   },
 
   host: computed(function() {
```
Comment (kellyselden) [inline app/services/fastboot.js]: Should this entire computed property get removed? or underscored?

Comment (kratiahuja) [inline app/services/fastboot.js]: No the `host` property is public API: http://ember-fastboot.com/docs/user-guide#request-host

Comment (kellyselden) [inline app/services/fastboot.js]: That refers to `fastboot.request.host`. This is `fastboot.host` if I'm not mistaken.

Comment (kratiahuja) [inline app/services/fastboot.js]: ohoo yes. need to delete that line.

**Comment by rahulk94** [comment]
Drive by comment: this PR is pretty old and looks like its been superseded by https://github.com/ember-fastboot/ember-cli-fastboot/pull/820/files. Probably worth closing.

**Pull request closed without merging**